examples/webgl_physics_volume.html

```diff
@@ -190,60 +191,19 @@
 
 			function processGeometry( bufGeometry ) {
 
-				// Obtain a Geometry
-				var geometry = new THREE.Geometry().fromBufferGeometry( bufGeometry );
+				// Ony consider the position values when merging the vertices
+				var posOnlyBufGeometry = new THREE.BufferGeometry();
+				posOnlyBufGeometry.addAttribute( 'position', bufGeometry.getAttribute( 'position' ) );
+				posOnlyBufGeometry.setIndex( bufGeometry.getIndex() );
 
-				// Merge the vertices so the triangle soup is converted to indexed triangles
-				geometry.mergeVertices();
+                // Merge the vertices so the triangle soup is converted to indexed triangles
+				var indexedBufferGeom = THREE.BufferGeometryUtils.mergeVertices( posOnlyBufGeometry );
 
-				// Convert again to BufferGeometry, indexed
-				var indexedBufferGeom = createIndexedBufferGeometryFromGeometry( geometry );
-
-				// Create index arrays mapping the indexed vertices to bufGeometry vertices
+                // Create index arrays mapping the indexed vertices to bufGeometry vertices
```

Comment:
nit: i think this got converted from tabs to spaces

Reply:
Yep. Line 199 too.

Reply:
Thanks I'll get these when I get home today

Reply:
Done! Let me know if there's anything else!